activerecord/test/cases/collection_cache_key_test.rb

```diff
@@ -18,7 +18,7 @@ class CollectionCacheKeyTest < ActiveRecord::TestCase
       developers = Developer.where(name: "David")
       last_developer_timestamp = developers.order(updated_at: :desc).first.updated_at
 
-      assert_match /\Adevelopers\/query-(\h+)-(\d+)-(\d+)\Z/, developers.cache_key
+      assert_match (/\Adevelopers\/query-(\h+)-(\d+)-(\d+)\Z/), developers.cache_key
 
```

Comment:
Can you remove the space after assert_matches? Also the parens look off, shouldn't they enclose both arguments?